.gitlab-ci.yml

```diff
@@ -136,7 +136,6 @@ build:windows:
     - pip install virtualenv pywin32
     - virtualenv venv
     - venv/Scripts/activate.ps1
-    - pip install pip==19.3.1; $true # $true ignores errors. need this to get the correct coincurve wheel. see commit notes for details.
```

Comment:
https://github.com/lbryio/lbry-sdk/commit/810a8e76d1f53305d6d7f19f43a2aaec765ec6c6

This was never necessary btw